src/os/ui/draw/draw.js

```diff
@@ -1,8 +1,143 @@
 goog.provide('os.ui.draw');
 
+goog.require('ol.Feature');
+goog.require('ol.geom.Polygon');
+goog.require('ol.style.Fill');
+goog.require('ol.style.Stroke');
+goog.require('ol.style.Style');
+goog.require('os.geo.jsts.OLParser');
+goog.require('os.style.area');
+
 
 /**
  * The menu to display when drawing interaction completes.
  * @type {os.ui.menu.Menu|undefined}
  */
 os.ui.draw.MENU = undefined;
+
+
+/**
+ * Build a list of features to represent the search grid that intersects with the drawn geometry
+ *
+ * @param {ol.Feature} feature The source feature to trace.
+ * @param {osx.ui.draw.GridOptions} options Config object for grid generation
+ * @return {Array.<ol.Feature>}
+ *
+ * @suppress {accessControls} To allow direct access to feature metadata.
+ */
+os.ui.draw.getGridFromFeature = function(feature, options) {
+  var valid = function(options) {
+    var b = (typeof options != 'undefined' && options != null);
```

Comment:
You did not leave `options` as optional in your param list, nor did you specify that it could be `null` or `undefined`, so this is unnecessary.

A better check for neither null or undefined is just `thing != null` (using type coercion). If you want to check specifically for one or the other, use strict equality: `thing === undefined` or `thing === null`. `typeof` should be avoided where possible.